### ISSUE TITLE
The list of snapshots for Linux seems to be missing quite a few entries

### ISSUE THREAD

**Issue by sharplet** [kylef/swiftenv-api#43]
This query returns a healthy list of snapshots for macOS:

```
curl -H 'Accept: text/plain' 'https://swiftenv-api.fuller.li/versions?snapshot=true&platform=osx
```

But far fewer are returned for Linux:

```
curl -H 'Accept: text/plain' 'https://swiftenv-api.fuller.li/versions?snapshot=true&platform=ubuntu16.04'
```

Notably:

- No Swift 3.x release snapshots are included
- Only a single 4.0-DEVELOPMENT snapshot is included
- No DEVELOPMENT snapshots from 2017 are included

**Comment by ikesyo** [comment]
Just submitted a fix #44.

**Comment by sharplet** [comment]
Thank you!
